package.json

```diff
@@ -21,7 +21,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@workos-inc/node": "7.21.0",
+    "@workos-inc/node": "7.26.0",
```

Comment:
Should this be a Major release since it includes breaking changes?

Reply:
Since authkit-nextjs is pre v1.0 it's okay to have breaking changes in minor updates. See [point 4](https://semver.org/) on semver.

Also that line just bumps the version of workos-node, it's not the release version for this library.